src/Components/EmailNotificationForm/EmailNotificationForm.js

```diff
@@ -0,0 +1,271 @@
+      { value: 'ga', label: 'Georgia' },
+      { value: 'hi', label: 'Hawaii' },
+      { value: 'id', label: 'Idaho' },
+      { value: 'il', label: 'Illinois' },
+      { value: 'in', label: 'Indiana' },
+      { value: 'ia', label: 'Iowa' },
+      { value: 'ks', label: 'Kansas' },
+      { value: 'ky', label: 'Kentucky' },
+      { value: 'la', label: 'Louisiana' },
+      { value: 'me', label: 'Maine' },
+      { value: 'md', label: 'Maryland' },
+      { value: 'ma', label: 'Massachusetts' },
+      { value: 'mi', label: 'Michigan' },
+      { value: 'mn', label: 'Minnesota' },
+      { value: 'ms', label: 'Mississippi' },
+      { value: 'mo', label: 'Missouri' },
+      { value: 'mt', label: 'Montana' },
+      { value: 'ne', label: 'Nebraska' },
+      { value: 'nv', label: 'Nevada' },
+      { value: 'nh', label: 'New Hampshire' },
+      { value: 'nj', label: 'New Jersey' },
+      { value: 'nm', label: 'New Mexico' },
+      { value: 'ny', label: 'New York' },
+      { value: 'nc', label: 'North Carolina' },
+      { value: 'nd', label: 'North Dakota' },
+      { value: 'oh', label: 'Ohio' },
+      { value: 'ok', label: 'Oklahoma' },
+      { value: 'or', label: 'Oregon' },
+      { value: 'pa', label: 'Pennsylvania' },
+      { value: 'ri', label: 'Rhode Island' },
+      { value: 'sc', label: 'South Carolina' },
+      { value: 'sd', label: 'South Dakota' },
+      { value: 'tn', label: 'Tennessee' },
+      { value: 'tx', label: 'Texas' },
+      { value: 'ut', label: 'Utah' },
+      { value: 'vt', label: 'Vermont' },
+      { value: 'va', label: 'Virginia' },
+      { value: 'wa', label: 'Washington' },
+      { value: 'wv', label: 'West Virginia' },
+      { value: 'wi', label: 'Wisconsin' },
+      { value: 'wy', label: 'Wyoming' }
+    ];
+
+    const options = [
+      <option className="state-name-option" name="state" value="Select" key="Select" disabled={true}>
+        Select
+      </option>
+    ];
+  
+    const stateOptions = states.map(state => {
+      return (
+        <option className="state-name-option" name="state" value={state.value} key={state.label}>
+          {state.label}
+        </option>
+      );
+    });
+
+    return options.concat(stateOptions);
+  }
+
+  render = () => {
+    return (
+      <form className="form-container">
+        <div className="form-header-container">
+          <p className="form-header">State Election Reminders</p>
+        </div>
+        <div className="form-content-container">
+          <div className="label-input-container">
+            <label className="label">
+              First Name<em>*</em>
+              <input type="text" 
+                name="first_name" 
+                value={this.state.first_name}
+                id="first_name" 
+                required="required" 
+                aria-required="true"
+                className="input"
+                onChange={(event) => this.handleInputChange(event)}/>
+            </label>
```

Comment:
Typically, I think that the `label` tag wraps **only** the text, and not the input itself, but I could definitely be wrong about this.

Reply:
On the topic of `label`, I believe it needs a way to tie the label to the thing it is labeling.  I believe in JSX, the attribute is `htmlFor`.  [Source](https://stackoverflow.com/a/22752418)

Reply:
I've refactored this so that all of the labels are on their own and have a htmlFor attribute.